test-engineering/contract-tests/client/tests/test_merino.py

```diff
@@ -69,7 +70,8 @@ def assert_200_response(
 
         if "wiki_fruit" in suggestion.provider:
             # The icon URL is static for WikiFruit suggestions
-            expected_suggestion = expected_suggestions_by_id[suggestion_id(suggestion)]
+            expected_suggestion = expected_suggestions_by_id[suggestion_id(
+                suggestion)]
```

Comment:
It's not documented, but we are using 'black' to format. Perhaps give that a run, since I think it will impact this change.